tests/view_tests/tests/test_debug.py

```diff
@@ -1274,6 +1274,14 @@ def test_cleanse_setting_recurses_in_dictionary(self):
             {'login': 'cooper', 'password': reporter_filter.cleansed_substitute},
         )
 
+    def test_cleanse_setting_recurses_in_dictionary_with_non_string_key(self):
+        reporter_filter = SafeExceptionReporterFilter()
+        initial = {1: {'login': 'cooper', 'password': 'secret'}}
```

Comment:
I think an example with `tuple` would be more realistic, e.g. `{('localhost', 8000): {....}`.

Reply:
Fine by me, my usecase is an `int` key but your example looks good too.